app/assets/javascripts/blob/upload_file.js.coffee

```diff
@@ -0,0 +1,35 @@
+Dropzone.options.blobFileDropzone = false;
+
+$(document).on "page:load", ->
+      $("#blobFileDropzone").dropzone()
+
+Dropzone.options.blobFileDropzone = {
+  autoProcessQueue: false,
+  clickable: true,
+  uploadMultiple: false,
+  paramName: "file",
+  parallelUploads: 1,
+  maxFiles: 1,
+  addRemoveLinks: true,
+  previewsContainer: '.dropzone-previews',
```

Comment:
@stanhu I added a border locally, as the container is also in the form with dropzone class, then it looks like 
![border](https://cloud.githubusercontent.com/assets/7370334/9837647/75c7ad4c-5a46-11e5-89c9-4ca7de0ef180.png)